metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/HiveConnectorFastPartitionService.java

```diff
@@ -169,6 +169,7 @@ private void createLocationForPartitions(final QualifiedName tableQName,
     private void createLocationForPartition(final QualifiedName tableQName,
         final PartitionInfo partitionInfo, final Table table, final boolean doFileSystemCalls) {
         String location = partitionInfo.getSerde().getUri();
+        Path path = null;
```

Comment:
Shouldn't this be final path? and is there any way through the if/else chain that doesn't result in an exception being thrown that this condition could even happen?

Reply:
Any runtime exception will end up failing the API call. We catch the checked exceptions and throw a relevant Metacat exception.